JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeScreens.kt

```diff
@@ -558,6 +543,108 @@ private fun PostListDivider() {
     )
 }
 
+/**
+ * Expanded search UI
+ */
+@Composable
+private fun HomeSearch(modifier: Modifier = Modifier) {
+    Surface(
+        shape = RoundedCornerShape(8.dp),
+        border = BorderStroke(Dp.Hairline, MaterialTheme.colors.onSurface.copy(alpha = .6f)),
+        elevation = 4.dp,
+        modifier = modifier.fillMaxWidth()
+    ) {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = Modifier.padding(horizontal = 8.dp)
+        ) {
+            CompositionLocalProvider(LocalContentAlpha provides ContentAlpha.medium) {
+                IconButton(onClick = { /* Functionality not supported yet */ },) {
+                    Icon(imageVector = Icons.Filled.Search, contentDescription = null)
```

Comment:
Nit: We should probably avoid a `null` content description here

Reply:
Added thanks!